DEV-GUIDE.md

```diff
@@ -1,5 +1,11 @@
 # Simperby Development Guide
 
+## Checklist
+
+- run `cargo fmt`
+- run `cargo clippy --all --all-targets --all-features`
+- run `cargo test --all --all-targets --all-features`
```

Comment:
why not the same as the commands in the ci?

Reply:
`-D warnings` is for emitting an error if failed, to notify the CI.
For `cargo test`, I updated the CI.